src/main/java/org/hyperledger/aries/webhook/WebSocketEventHandler.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2020-2022 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/acapy-java-client
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.aries.webhook;
+
+public class WebSocketEventHandler extends EventHandler {
```

Comment:
Shouldn't clients be extending this class? On its own and empty it does not make much sense to me having that one here.